### PR TITLE
CSI-1208 bugfix display gas unit and disable decimal gas values for btc forks

### PR DESCRIPTION
**Pull request by TejasvCypherock**

### CODE REVIEW

**Comment by irshadCypherock** [comment]
Task linked: [CSI-1208 Bugfix: display gas unit and disable decimal gas values for BTC forks](https://app.clickup.com/t/37308523/CSI-1208)

File: cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx [hunk 1]
```diff
@@ -602,12 +607,15 @@ const Recipient: React.FC<StepComponentProps> = props => {
...
-            COINS[coinDetails.slug.toLowerCase()].fees
-          }`}
+          placeHolder="Enter transaction fees"
           onChange={handleTransactionFeeChange}
           type="number"
           value={transactionFee}
+          customIcon={
+            <Typography className={classes.feeUnit}>
+              {COINS[coinDetails.slug.toLowerCase()]?.fees}
+            </Typography>
```
Comment (HarishRocks) [inline cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx:617]: Pls amend
"sat per byte" to "sat/byte" in comm module

File: cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx [hunk 2]
```diff
@@ -602,12 +607,15 @@ const Recipient: React.FC<StepComponentProps> = props => {
...
+          }
```
Comment (HarishRocks) [inline cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx:618]: Pls amend "sat per byte" to "sat/byte" in comm module

Comment (HarishRocks) [reply cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx:618]: this property of CoinData is only used on UI, so safe to change it

**Review by HarishRocks — CHANGES_REQUESTED** [review]

File: cysync/src/pages/mainApp/sidebar/wallet/send/stepperForm.tsx [hunk 3]
```diff
@@ -584,11 +585,17 @@ const SendForm: React.FC<StepperProps> = ({ stepsData, handleClose }) => {
...
 
   const handleTransactionFeeChange = (e: any) => {
-    setTransactionFee(e.target.value);
+    setTransactionFee(
+      e.target.value
+        ? isBtcFork
+          ? Math.round(e.target.value)
+          : e.target.value
+        : e.target.value
+    );
```
Comment (HarishRocks) [inline cysync/src/pages/mainApp/sidebar/wallet/send/stepperForm.tsx:594]: Dont do it like this
declare a new prop on CustomSlider that takes step and pass it here

**Review by HarishRocks — CHANGES_REQUESTED** [review]

File: cysync/src/pages/mainApp/sidebar/wallet/send/stepperForm.tsx (hunk 3 above)
Comment (HarishRocks) [reply cysync/src/pages/mainApp/sidebar/wallet/send/stepperForm.tsx:594]: we need to think of a good solution that handles input and slider fields optimally

**Pull request merged**